Libplanet.Action/ActionEvaluator.cs

```diff
@@ -34,18 +36,23 @@ public class ActionEvaluator : IActionEvaluator
         /// at the end for each <see cref="IPreEvaluationBlock"/> that gets evaluated.</param>
         /// <param name="blockChainStates">The <see cref="IBlockChainStates"/> to use to retrieve
         /// the states for a provided <see cref="Address"/>.</param>
+        /// <param name="keyValueStore">
+        /// A <see cref="IKeyValueStore"/> implementation to use for storing the states.
+        /// </param>
         /// <param name="actionTypeLoader"> A <see cref="IActionLoader"/> implementation using
         /// action type lookup.</param>
         public ActionEvaluator(
             PolicyBlockActionGetter policyBlockActionGetter,
             IBlockChainStates blockChainStates,
+            IKeyValueStore keyValueStore,
```

Comment:
Rather icky. In any case, does this need to be introduced at this moment? Think this might be an artifact from splitting #3378. 🙄

Reply:
No, it is from #3378 as you know. Might be not perfectly split from the previous PR...